.gitignore

```diff
@@ -34,3 +34,5 @@ src/editor/EditorBuild/
 test/projects/ProfilingTest/Platformer/.vscode/settings.json
 
 .vscode/settings.json
+
+playground.jl
```

Comment:
I took the liberty to add a scratch file I am using to the .gitignore. I hope that's okay!